pathd/path_zebra.c

```diff
@@ -30,13 +31,93 @@ struct zclient *zclient;
 static struct zclient *zclient_sync;
 
 /* Global Variables */
-bool g_has_router_id_v4 = false;
-bool g_has_router_id_v6 = false;
+bool g_has_router_id_v4;
```

Comment:
why is it needed to modify those global variables ?

Reply:
fixed